cms/utils/conf.py

```diff
@@ -67,6 +67,7 @@ def wrapper():
     'WIZARD_DEFAULT_TEMPLATE': constants.TEMPLATE_INHERITANCE_MAGIC,
     'WIZARD_CONTENT_PLUGIN': 'TextPlugin',
     'WIZARD_CONTENT_PLUGIN_BODY': 'body',
+    'WIZARD_CONTENT_PLACEHOLDER': None,  # Use first placeholder it finds.
```

Comment:
would it be possible to define 'content' as default and if it doesn't find content, or any provided string, fall back to the first?